app/src/main/java/com/example/taskyy/ui/events/AgendaEvent.kt

```diff
@@ -24,4 +25,5 @@ sealed interface AgendaEvent {
     data class EditExistingReminder(val agendaItem: AgendaEventItem) : AgendaEvent
     data class DeleteExistingReminder(val agendaEventItem: AgendaEventItem) : AgendaEvent
     data class IsEllipsisMenuExpanded(val isEllipsisMenuExpanded: Boolean) : AgendaEvent
+    data class StartWorkManager(val context: Context) : AgendaEvent
```

Comment:
Why do you need to trigger scheduling this from the UI? You can do this from the `init` in your ViewModel

Reply:
I triggered it from the UI so that I had a reference to the context. I need the context in the repository to launch the worker. Is there a better way to get the context to the work manager?